steps/28/README.md

```diff
@@ -131,66 +131,33 @@ As you can see, the test case reads like a user story, we actually do not need t
-
-// import all your integration tests here
-void Promise.all([
-	import("sap/ui/core/Core"), // required to wait until Core has booted to start the QUnit tests
-	import("ui5/walkthrough/test/integration/NavigationJourney"),
-]).then(([{default: Core}]) => Core.ready()).then(() => {
-	QUnit.start();
-});
+import "./NavigationJourney";
 ```
 
 ***
 
-### webapp/test/integration/opaTests.qunit.html \(New\)
-
-Finally we create a new `opaTests.qunit.html` page under `webapp/test/integration`.
-
-This HTML page contains our test suite for all OPA tests of the app. We use the same namespace as for our application.
-
-Then we load the basic QUnit functionality via script tags from OpenUI5 so that we can execute the test journey. Finally we load the `opaTests.qunit`, which then again loads our `NavigationJourney`.
-
-```html
-<!DOCTYPE html>
-<html>
-<head>
-	<title>UI5 TypeScript Walkthrough - Integration Tests</title>
-	<meta charset="utf-8">
-
-	<script
-		id="sap-ui-bootstrap"
-		src="../../resources/sap-ui-core.js"
-		data-sap-ui-theme="sap_horizon"
-		data-sap-ui-resourceroots='{
-			"ui5.walkthrough": "../../"
-		}'
-		data-sap-ui-animation="false"
-		data-sap-ui-compat-version="edge"
-		data-sap-ui-async="true">
-	</script>
-
-	<link rel="stylesheet" type="text/css" href="../../resources/sap/ui/thirdparty/qunit-2.css">
-
-	<script src="../../resources/sap/ui/thirdparty/qunit-2.js"></script>
-	<script src="../../resources/sap/ui/qunit/qunit-junit.js"></script>
-
-	<script src="./opaTests.qunit.js"></script>
-</head>
-<body>
-	<div id="qunit"></div>
-	<div id="qunit-fixture"></div>
-</body>
-</html>
+### webapp/test/testsuite.qunit.ts
+
+Finally we reference the new `integration/opaTests.qunit.ts` in the `testsuite.qunit.ts` file. The `.qunit.ts` extension is omitted and will be added automatically during runtime.
+
+```ts
+export default {
+  // ...
+	tests: {
+		"unit/unitTests": {
+			title: "UI5 TypeScript Walkthrough - Unit Tests"
+		},
+		"integration/opaTests": {
+			title: "UI5 TypeScript Walkthrough - Integration Tests"
+		}
+	}
+};
 ```
 
-When you call the `webapp/test/integration/opaTests.qunit.html` page of your project on the server, you should see the QUnit layout and a test “Should see the Hello dialog” is executed immediately. It will load the app component on the right side of the page. There you can see what operations the test is performing on the app, if everything works correctly the button click is triggered, then a dialog is shown and the test case is green.
+If we now open the `webapp/test/testsuite.qunit.html` file in the browser and select select `integration/opaTests`, we should see the QUnit layout and a test “Should see the Hello dialog” is executed immediately. It will load the app component on the right side of the page. There you can see what operations the test is performing on the app, if everything works correctly the button click is triggered, then a dialog is shown and the test case is green.
 
```

Comment:
If we now open the `webapp/test/testsuite.qunit.html` file in the browser and select `integration/opaTests`, the QUnit layout should appear and a test “Should see the Hello dialog” will run immediately. This action will load the app component on the right side of the page. There you can see the operations the test is performing on the app. If everything works correctly, a button click will be triggered, then a dialog will be displayed and the test case will be green.